src/components/global/Playground/stackblitz.utils.ts

```diff
@@ -54,20 +54,25 @@ const openHtmlEditor = async (code: string, options?: EditorOptions) => {
       options?.includeIonContent ? 'html/index.withContent.html' : 'html/index.html',
       'html/variables.css',
       'html/package.json',
+      'html/tsconfig.json',
+      'html/vite.config.ts',
     ],
     options.version
   );
 
+  const package_json = JSON.parse(defaultFiles[3]);
+
   const indexHtml = 'index.html';
   const files = {
+    'package.json': JSON.stringify(package_json, null, 2),
     'index.ts': defaultFiles[0],
     [indexHtml]: defaultFiles[1],
     'theme/variables.css': defaultFiles[2],
```

Comment:
Originally I moved `index.ts` and `theme/variables.css` under `src/` like the other frameworks but if we did that we would have to update all of the playgrounds where these files are overridden like:

https://github.com/ionic-team/ionic-docs/blob/d9ffaf68d2865af1926536a75ccd6d5283a3f52d/static/usage/v8/theming/always-dark-mode/index.md?plain=1#L23-L25

I can do this if desired but this would change ~80 files.